llvm/lib/Object/IRSymtab.cpp

```diff
@@ -54,6 +54,10 @@ static const char *PreservedSymbols[] = {
     // TODO: Are there similar such variables?
     "__ssp_canary_word",
     "__stack_chk_guard",
+    // The dead store elimination pass can fold malloc + memset calls into
+    // a single call to calloc. Prevent thin LTO from marking calloc a dead
```

Comment:
// Prevent thin LTO from marking calloc a dead function ...

I think this sentence can be removed, as it describes the problem that applies too all preserved symbols, which can use a comment applying to the whole variable.

This change affects regular LTO, so "thin LTO" is not accurate. This list also prevents internalization.

Reply:
I think this comment is not addressed.